src/screens/DiscoveryScreenV2/elements/DiscoveryItemList.js

```diff
@@ -117,6 +118,19 @@ const DomainList = (props) => {
                 {item.description ? item.description : ''}
               </Text>
             )}
+            {item.comumnityInfo?.length > 0 && !item.user_id_follower && item.isUser && (
+              <View style={styles.communityTextContainer} testID="communityDesc">
+                <Text style={styles.textProfileFullName} numberOfLines={1} ellipsizeMode="tail">
+                  Also in{' '}
+                  <Text style={styles.communityText}>
+                    {`${item.comumnityInfo
+                      .slice(0, 3)
+                      .map((community) => `#${community}`)
+                      .join(', ')}`}
+                  </Text>
+                </Text>
+              </View>
+            )}
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

There seems to be a typo in the property name `comumnityInfo`. If this is not intentional, it should be corrected to `communityInfo`.
```diff
- {item.comumnityInfo?.length > 0 && !item.user_id_follower && item.isUser && (
+ {item.communityInfo?.length > 0 && !item.user_id_follower && item.isUser && (
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->